Makefile

```diff
@@ -149,7 +149,8 @@ admission: init
 adm_image: admission
 	@echo "building admission controller docker images"
 	@cp ${ADMISSION_CONTROLLER_BIN_DIR}/${POD_ADMISSION_CONTROLLER_BINARY} ./deployments/image/admission
-	docker build ./deployments/image/admission -t ${REGISTRY}/yunikorn:admission-${VERSION}
+	docker build ./deployments/image/admission -t ${REGISTRY}/yunikorn:admission-${VERSION} \
+	  --build-arg DOCKER_IMAGE_REGISTRY=${REGISTRY} DOCKER_IMAGE_VERSION=${VERSION}
```

Comment:
It seems that --build-arg can only support one argument so that we can add another "--build-arg" before "DOCKER_IMAGE_VERSION=${VERSION}".

Reply:
Ah, good catch. I only tested 1 arg I guess. Let me fix this, thanks!